src/font/text.cpp

```diff
@@ -223,18 +215,12 @@ point pango_text::get_cursor_position(const unsigned column, const unsigned line
 {
 	this->recalculate();
 
-	// First we need to determine the byte offset, if more routines need it it
-	// would be a good idea to make it a separate function.
+	// Determing byte offset
```

Comment:
I just want to note that I had no issue with the other comments in this function. I still have no issue with them though.

Reply:
Okay, I can add them back.

Reply:
I'm only mentioning it in case you removed them because you thought I didn't like them. If you removed them because _you_ didn't like them, or because they're no longer accurate, then I don't mind.

Reply:
Okay, thanks for clarifying.